### DISCUSSION TITLE
[Localization Problem] Incorrect Mission Type Description

### DISCUSSION THREAD

**Discussion by VeesTM** [FakeFishGames/Barotrauma#14665] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Mission type: Enemy submarine
English version description:
![image](https://github.com/user-attachments/assets/da9af913-acb0-42f9-ab0d-fe72bc2f0e86)

Russian version description:
![image](https://github.com/user-attachments/assets/b1f0b4df-fe5d-46e8-b124-ec89da0397bf)

In Russian version there is a mistake. Written "...уничтожьте есго вместе..." instead "...уничтожьте его вместе...".

### Reproduction steps

1. Switch the game to the Russian language pack
2. Host server
3. Switch game mode to mission
4. Look at description mission Enemy submarine

### Bug prevalence

Just once

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
This has been addressed since the report was created, looks like we just forgot to close the ticket. :)
